README.md

```diff
@@ -51,9 +51,24 @@ async function main() {
           id: 3
       });
 
-  if (linkuriousConfigurationResponse.isSuccess() && visualizationResponse.isSuccess()) {
-      const ogmaConfiguration = linkuriousConfigurationResponse.body.ogma;
 
+  const nodeTypesResponse = await this.rc.graphSchema.getTypesWithAccess({
+    entityType: 'node'
+  });
+  const edgeTypesResponse = await this.rc.graphSchema.getTypesWithAccess({
+    entityType: 'edge'
+  });
+  
```

Comment:
there are several double empty lines, I don't know if done on purpose or by mistake (usually one should be enough to separate the code).